config.go

```diff
@@ -1608,17 +1607,17 @@ func extractBitcoindRPCParams(networkName string,
 		dataDir = string(dataDirSubmatches[1])
 	}
 
-	chainDir := "/"
+	chainDir := ""
```

Comment:
Can set this to `chainDir` directly and only set to empty on `case "mainnet"`?

Reply:
``` go

	chainDir := ""
	switch networkName {
	case "mainnet":
		chainDir = ""
	case "testnet3":
		chainDir = "testnet3"
	case "testnet4":
		chainDir = "testnet4"
	case "regtest":
		chainDir = "regtest"
	default:
		return "", "", "", "", fmt.Errorf("unexpected networkname %v", networkName)
	}

```

hello,
I'm not sure if there are other networks, have they met expectations?